### DISCUSSION TITLE
How to combine editor and uploader?

### DISCUSSION THREAD

**Discussion by DaniGuardiola** [uploadcare/file-uploader#331] [Q&A] [answered]
It is possible to allow for editing (e.g. cropping) as part of the upload experience (before actually uploading)?

Thanks in advance!

**Comment by nd0ut** [comment, marked as answer]
Hey @DaniGuardiola, 

So far, we have Cloud Image Editor only, which is enabled by default so you could edit already uploaded images. 
You can't edit images before uploading them yet. But we have plans to make a Local Image Editor, with which it will be possible to edit images before they are uploaded.

See how it works - https://v.usetapes.com/0pbasFNzoN

**Reply by DaniGuardiola** [reply]
Thanks! Looking forward to it :)
